src/app.rs

```diff
@@ -351,6 +352,10 @@ fn usage() -> HashMap<&'static str, Help> {
         , "Separate results by the null character"
         , "Separate search results by the null character (instead of newlines). Useful for \
            piping results to 'xargs'.");
+    doc!(h, "prune"
+        , "Do not descend into matched directories"
+        , "When a directory matches the pattern, fd does not descend into it but still shows \
+           all matches at the same level as that directory.");
```

Comment:
To me, this is slightly ambiguous for people who aren't familiar with find's -prune option. In particular, I think it should be more explicitly stated that the matched directories are included in the output. Maybe something like:

"Do not descend into matched directories"
"When a directory matches the pattern, fd does not descend into it. The directory itself is not excluded."